src/analytics/ValoraAnalytics.ts

```diff
@@ -129,21 +125,6 @@ class ValoraAnalytics {
     } catch (error) {
       Logger.error(TAG, `Segment setup error: ${error.message}\n`, error)
     }
-
-    try {
-      const statsigUser = getDefaultStatsigUser()
-      // getAnonymousId causes the e2e tests to fail
-      const overrideStableID = isE2EEnv ? E2E_TEST_STATSIG_ID : await Analytics.getAnonymousId()
```

Comment:
I guess this was originally here because we assumed getting anonymousId should be after initializing segment, but this may not be true. If we don't need it and if segment consistently returns the ID before / after initialization, I guess we can also revert adding ValoraAnalytics.init() to AppInitGate and do it async as it was prior to this commit https://github.com/valora-inc/wallet/commit/5a982019f89ef552c8fc9e5f1b0fee7bdedd7c19